src/pointfree/project.js

```diff
@@ -0,0 +1,14 @@
+/** @license ISC License (c) copyright 2016 original and current authors */
+/** @author Karthik Iyengar (karthikiyengar) */
+
+const isFunction = require('../core/isFunction')
+
+function project(m) {
```

Comment:
This will also need to take an Integer as its first argument, but we should only throw if it is not an Integer, no need to check the range between 1-10, we can let the underlying `m` handle that.

Reply:
Could you please elaborate? I do not see any checks here except the `isFunction` check - everything else is indeed being deferred to the underlying `m`. Are you suggesting that we add another integer check here?

Reply:
So how do we pass in WHICH element we want?
`m.project` take a number 1-10 yeah?

Reply:
Hah, sorry - totally missed that. I shall add the integer!